restdocs-openapi-gradle-plugin/src/main/kotlin/com/epages/restdocs/openapi/gradle/OpenApi20Generator.kt

```diff
@@ -110,28 +115,37 @@ internal object OpenApi20Generator {
         return if(securityRequirements.type.equals(SecurityType.OAUTH2) && securityRequirements.requiredScopes != null) securityRequirements.requiredScopes else listOf()
     }
 
-    private fun pathParameterDescriptor2PathParameter(parameterDescriptor: ParameterDescriptor): PathParameter {
+    private fun pathParameterDescriptor2Parameter(parameterDescriptor: ParameterDescriptor): PathParameter {
         return PathParameter().apply {
             name = parameterDescriptor.name
             description = parameterDescriptor.description
             type = parameterDescriptor.type.toLowerCase()
         }
     }
 
-    private fun requestParameterDescriptor2PathParameter(parameterDescriptor: ParameterDescriptor): QueryParameter {
+    private fun requestParameterDescriptor2Parameter(parameterDescriptor: ParameterDescriptor): QueryParameter {
         return QueryParameter().apply {
             name = parameterDescriptor.name
             description = parameterDescriptor.description
             type = parameterDescriptor.type.toLowerCase()
         }
     }
 
+    private fun header2Parameter(headerDescriptor: HeaderDescriptor): HeaderParameter {
+        return HeaderParameter().apply {
+            name = headerDescriptor.name
+            description = headerDescriptor.description
+            type = headerDescriptor.type.toLowerCase()
+        }
+    }
+
     private fun responseModel2Response(responseModel: ResponseModel): Response {
         return Response().apply {
             description = ""
+            headers = responseModel.headers
+                .map { it.name to PropertyBuilder.build(it.type, "", mapOf()).description(it.description) }
```

Comment:
Having a hard time understanding the need for `PropertyBuilder.build(it.type, "", mapOf())`, could also probably be simplified to `PropertyBuilder.build(it.type, null, null)`